Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/CalculateSelfScatteringCorrection.py

```diff
@@ -0,0 +1,113 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2019 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+from __future__ import (absolute_import, division, print_function)
+
+from mantid.simpleapi import (CalculatePlaczekSelfScattering, ConvertToDistribution, ConvertUnits, CreateWorkspace,
+                              DeleteWorkspace, DiffractionFocussing, Divide, ExtractSpectra, FitIncidentSpectrum,
+                              LoadCalFile, SetSample)
+from mantid.api import (DataProcessorAlgorithm, AlgorithmFactory, WorkspaceProperty)
+from mantid.kernel import Direction
+import numpy as np
+
+
+class CalculateSelfScatteringCorrection(DataProcessorAlgorithm):
+
+    def __init__(self):
```

Comment:
I think we can get rid of the `__init__` here as Python should do this by itself.